main/FatesHistoryInterfaceMod.F90

```diff
@@ -253,8 +253,13 @@ module FatesHistoryInterfaceMod
   integer :: ih_seed_bank_si            ! carbon only
   integer :: ih_seeds_in_si             ! carbon only
   integer :: ih_seeds_in_local_si       ! carbon only
-  integer :: ih_ungerm_seed_bank_si        ! carbon only
-  integer :: ih_seedling_pool_si    ! carbon only
+  integer :: ih_ungerm_seed_bank_si     ! carbon only
+  integer :: ih_seedling_pool_si        ! carbon only
+  integer :: ih_seed_bank_si_pft          ! carbon only
```

Comment:
These could be moved down to Line 572 where the rest of the site x pft variables are defined

Reply:
Good recommendation @JessicaNeedham.  I've made this change.